tests/test_updater_root_rotation_integration.py

```diff
@@ -342,22 +415,30 @@ def test_root_rotation_missing_keys(self):
     shutil.copytree(os.path.join(self.repository_directory, 'metadata.staged'),
                     os.path.join(self.repository_directory, 'metadata'))
 
-    try:
+    with self.assertRaises(tuf.exceptions.NoWorkingMirrorError) as cm:
       self.repository_updater.refresh()
 
-    except tuf.exceptions.NoWorkingMirrorError as exception:
-      for mirror_url, mirror_error in six.iteritems(exception.mirror_errors):
-        url_prefix = self.repository_mirrors['mirror1']['url_prefix']
-        url_file = os.path.join(url_prefix, 'metadata', '2.root.json')
+    for mirror_url, mirror_error in six.iteritems(cm.exception.mirror_errors):
+      url_prefix = self.repository_mirrors['mirror1']['url_prefix']
+      url_file = os.path.join(url_prefix, 'metadata', '2.root.json')
 
-        # Verify that '2.root.json' is the culprit.
-        self.assertEqual(url_file.replace('\\', '/'), mirror_url)
-        self.assertTrue(isinstance(mirror_error,
+      # Verify that '2.root.json' is the culprit.
+      self.assertEqual(url_file.replace('\\', '/'), mirror_url)
```

Comment:
I know this is how most tests do this but replacing the 3 lines with this should do the job and makes the test more readable:
`self.assertTrue(mirror_url.endswith("/2.root.json"))`

Reply:
Thanks for this suggestion, I've pushed a patch implementing it: adbb5fe